src/theory/strings/inference_manager.h

```diff
@@ -125,16 +126,22 @@ class InferenceManager
    * If the flag asLemma is true, then this method will send a lemma instead
    * of an inference whenever applicable.
    */
-  void sendInference(std::vector<Node>& exp,
-                     std::vector<Node>& exp_n,
+  void sendInference(const std::vector<Node>& exp,
+                     const std::vector<Node>& exp_n,
                      Node eq,
                      const char* c,
                      bool asLemma = false);
   /** same as above, but where exp_n is empty */
-  void sendInference(std::vector<Node>& exp,
+  void sendInference(const std::vector<Node>& exp,
                      Node eq,
                      const char* c,
                      bool asLemma = false);
+  /** Send inference
+   *
+   * Calls the appropriate call to send inference based on the infer info
```

Comment:
"Calls the appropriate call" reads a bit weird. Maybe just refer to the documentation of `sendInference`?